### DISCUSSION TITLE
Invite Member button is not working

### DISCUSSION THREAD

**Discussion by MuhammadFaisal1521** [community/community#23354] [New to GitHub]
<p>Here I am invite my fellows to our organization. after selecting his ID, Invite button still disable or not working. Any one have any idea what is wrong with it<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/e/9e1c88ede5805483fa4997b52b2309e36a776608.png" data-download-href="/uploads/short-url/myIzIfWhGMD3j2kup4T32hpWapO.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25382318/181076492-0ca0d4a0-eea7-4e52-ab5f-cd650dde4965.png" alt="image" data-base62-sha1="myIzIfWhGMD3j2kup4T32hpWapO" width="690" height="321" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e1c88ede5805483fa4997b52b2309e36a776608_2_690x321.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e1c88ede5805483fa4997b52b2309e36a776608_2_1035x481.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/e/9e1c88ede5805483fa4997b52b2309e36a776608.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e1c88ede5805483fa4997b52b2309e36a776608_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1356×631 39.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="MuhammadFaisal1521" data-post="1" data-topic="192742">
<div class="title">
<div class="quote-controls"></div>
 MuhammadFaisal1521:</div>
<blockquote>
<p>after selecting his ID, Invite button still disable or not working. Any one have any idea what is wrong with it</p>
</blockquote>
</aside>
<p>When the user shows up in the blue dropdown, you still have to click on the shown user for your choice to become effective, in fact you can see the <code>+</code> button in the proposed user — the fact that the screenshot is still showing the suggested user means it wasn’t picked yet.</p>

**Comment by adriennrichmond** [comment, marked as answer]
<p>This is commonly caused by browser extensions, as it was the case this time. Thank you for raising this!</p>
